source/schema/export.txt

```diff
@@ -24,156 +24,95 @@ If you have not already done so, analyze your schema:
-     - Name of this data type.
-   
-   * - ``types[n].bsonType``
-     - String
-     - |bson| type of this data type.
- 
-   * - ``types[n].path``
-     - String
-     - Path to the corresponding field within a document.
- 
-   * - ``types[n].count``
-     - Integer
-     - Number of times this data type appears in the corresponding 
-       field.
-   
-   * - ``types[n].values``
-     - Array
-     - Array of the actual sampled values that 
-       appear in the corresponding field and match this data type.
- 
-   * - ``types[n].total_count``
-     - Integer
-     - If the corresponding field is an array, the number of elements 
-       in that array.
-
-   * - ``types[n].probability``
-     - Number
-     - Probability that the value of the corresponding field is this 
-       data type in a random document.
- 
-   * - ``types[n].unique``
-     - Integer
-     - Number of unique values of this data type that appear in 
-       the corresponding field.
- 
-   * - ``types[n].has_duplicates``
-     - Boolean
-     - ``true`` if a single value of this data type appears multiple 
-       times in the corresponding field. Otherwise ``false``.
- 
-   * - ``types[n].lengths``
-     - Array
-     - If this data type is an array, an array of integers representing 
-       the lengths of arrays found in the corresponding field. Not 
-       present for other data types.
- 
-   * - ``types[n].average_length``
-     - Number
-     - If this data type is an array, the average length of arrays in 
-       the corresponding field across sampled documents. Not present 
-       for other data types.
- 
-   * - ``total_count``
-     - Integer
-     - Number of documents sampled from the collection.
- 
-   * - ``type``
-     - String or Array
-     - String or array of strings representing possible types for 
-       the corresponding field.
- 
-   * - ``has_duplicates``
-     - Boolean
-     - ``true`` if a single value appears multiple times in 
-       the corresponding field. Otherwise ``false``.
- 
-   * - ``probability``
-     - Number
-     - Probability that a random document contains the corresponding 
-       field.
+   Select the location on your filesystem to save your schema to.
+
+.. _export-format-standard:
+
+Standard Format Schema Object Properties
+-----------------------------------------
+
+.. include:: /includes/schema-export-format-standard.rst
+
+.. include:: /includes/addl-json-fields.rst
+
+.. _export-format-mongodb:
+
+MongoDB Format Schema Object Properties
+---------------------------------------
+
+.. include:: /includes/schema-export-format-mongodb.rst
+
+.. include:: /includes/addl-json-fields.rst
+
+.. _export-format-expanded:
+
+Expanded Format Schema Object Properties
+----------------------------------------
+
+.. include:: /includes/schema-export-format-expanded.rst
+
+.. include:: /includes/addl-json-fields.rst
+
+Limitations
+-----------
+
+Compass cannot export a schema that has more than 1000 distinct fields.
```

Comment:
[suggestion] We should use the `|compass-short|` snooty variable here, so in case this feature makes it out to Data Explorer, we can easily swap out the text through copybara. Same goes for the next instance on line 85!
```suggestion
|compass| cannot export a schema that has more than 1000 distinct fields.
```

Reply:
Done.